SwiftCharts/Chart.swift

```diff
@@ -788,11 +789,15 @@ open class ChartView: UIView, UIGestureRecognizerDelegate {
             
             chart.onPanEnd()
             
-        case .cancelled: break;
-        case .failed: break;
+        case .cancelled:
+            fallthrough
+        case .failed:
+            fallthrough
         case .possible:
 //            sender.state = UIGestureRecognizerState.Changed
-            break;
+            fallthrough
+        default:
```

Comment:
Let's make this `@unknown default:` such that the compiler emits a warning.

Reply:
I left in the .possible case due to the commented code. I'ff let you decide how to structure this switch statement going forward.